File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        outerLoop:
        for(int i=2;i<=n;i++){
            for(int j=2;j<=Math.sqrt(i);j++){
                if(i%j==0)
                    continue outerLoop;
            }
            answer++;
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.py
import math
def solution(n):
    test=[True]*(n+1)
    test[0], test[1]=False, False
    
    for i in range(2, int(math.sqrt(n))+1):
        if test[i]:
            for j in range(i*i,n+1,i):
                test[j]=False
    return sum(test)